src/renderer/browser/values/protocols.js

```diff
@@ -0,0 +1,11 @@
+export const customProtocols = {
```

Comment:
All our other `values` files export constants in uppercase.  Can we call these `PROTOCOLS~ and `CUSTOM_PROTOCOLS` for consistency?

Reply:
Ah right, forgot that 😄